build.gradle

```
@@ -7,10 +7,10 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.2'
+        classpath 'com.android.tools.build:gradle:7.3.0'
         classpath 'com.google.gms:google-services:4.3.14'
-        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10'
-        classpath 'androidx.navigation:navigation-safe-args-gradle-plugin:2.5.1'
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.20-RC'
```

We're not interested in this RC
```suggestion
        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10'
```